tests/system/python/e2e/test_e2e_notification_service_with_plugins.py

```diff
@@ -271,31 +278,28 @@ def enable_notification(self, foglamp_url, cat_name, is_enabled=True):
         r = conn.getresponse()
         assert 200 == r.status
 
-    def test_sent_and_receive_notification(self, foglamp_url, start_south, wait_time):
-        data = {"name": "Test4", "description": "Test4_Notification", "rule": NOTIFY_RULE, "channel": NOTIFY_PLUGIN,
-                "enabled": False, "notification_type": "retriggered"}
+    def sent_and_receive_notification(self, foglamp_url, start_south, wait_time):
```

Comment:
Please keep it as test. Though it has no asserting in the same block but  it has call to _verify_audit_log_entry with assertions